src/App.tsx

```diff
@@ -97,7 +83,7 @@ export const App: React.FC = () => {
             )}
           >
             <div className="tile is-child box is-success ">
-              {selectedPost && <PostDetails post={selectedPost} />}
+              {selectedPost && <PostDetails />}
```

Comment:
The conditional rendering of the 'PostDetails' component is correct. Ensure that the 'PostDetails' component is properly defined and imported.